turbopack/crates/turbopack-ecmascript/src/references/async_module.rs

```diff
@@ -76,7 +76,7 @@ impl OptionAsyncModule {
 }
 
 #[turbo_tasks::value(transparent)]
-struct AsyncModuleIdents(FxIndexSet<String>);
+struct AsyncModuleIdents(FxIndexSet<(String, u32)>);
```

Comment:
Create a new SyntaxContext Type Wrapper or add some documentation to explain this number